contribute/creating_package.md

```diff
@@ -16,12 +16,12 @@ In case of any issues, remember that [help](https://github.com/elizavetasemenova
 
```

Comment:
We embed this video: https://www.youtube.com/watch?v=QVmU29rCjaA on this page

Reply:
As @KristofferC pointed out above, the part on continuous integration from that video is not applicable anymore. also, the command to generate a package from the template now is slightly different (does not use  the command `generate` as in the video). Still embed, and mention that for the updated version it is better to follow this page? or leave as is, acknowledging the tutorial, but presenting the updated version of commands?

Reply:
Updated version of the commands and such, it is also worth thinking about, whether we just want to link to the getting started guides for these other resources. If we write code specific tutorials, they will inevitable become out dated vs if we just link to other things and provide commentary we are more future proof.

Reply:
With that said, we should still link to Chris's video and as you said, make the disclaimers. I can also add a note on that video